O/ONNXRuntime/build_tarballs.jl

```diff
@@ -0,0 +1,124 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ONNXRuntime"
+version = v"1.10.0"
+
+# Cf. https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements
+# Cf. https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#requirements
+cuda_tag = "11.3" # No CUDA 10.2, since pre-built x86_64 ONNXRuntime-CUDA binaries are built for CUDA 11
```

Comment:
If you want broad compatibility, you'll also want to build against other versions (e.g. if 11.3 is required, also 11.4, 11.5, etc).

Reply:
So the conclusion in https://github.com/JuliaPackaging/Yggdrasil/pull/4386#discussion_r864789688 that it was sufficient to build for one CUDA 10 version and one CUDA 11 version does not hold?

Reply:
Ah, right. I'm not actually sure, to be honest; all of the compatibility guarantees and documentation only applies to the interface with the driver (i.e. libcuda), and I'm not sure whether binaries that have been built against one specific toolkit's libraries are forwards compatible. We are assuming that kind of compatibility for pre-built libraries like CUDNN, so maybe it's fine to do that here as well. On the other hand, it doesn't hurt actually rebuilding for the different versions of CUDA that we support, right?

Reply:
Yes - that's also how I understood it, that the `cuda` tag relates to compatibility with the driver.

I've pushed (a WIP of) a JLL-adapted version of [platform selection based on augmentation tags](https://pkgdocs.julialang.org/v1.7/artifacts/#Extending-Platform-Selection) to https://github.com/jw3126/ONNXRunTime.jl/pull/19 . It queries the CUDA driver.

I would prefer to save the bits and just build one CUDA version per CUDA-major version.

Reply:
> Yes - that's also how I understood it, that the cuda tag relates to compatibility with the driver.

Right, but that's the part I think I want to avoid: https://github.com/JuliaPackaging/Yggdrasil/blob/b715966a830c6865d87f895bd2e4ad73d725b0b7/platforms/cuda.jl#L13-L16

Reply:
OK.

I chose 11.3 to have a TensorRT JLL for aarch64.

I gave #4679 a peek - I guess that would be ideal?

But any way, in this case, limiting support to 11.3 might (currently) be necessary due to TensorRT JLL (?)

Reply:
Yeah it's fine to limit this to 11.3 for now, since you'll be using these artifacts manually anyway. I just wanted to mention that we might have to update this recipe if I ever get to finishing the CUDA JLL stuff.

Reply:
OK - great :-)